04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+\chapter{Environment}
+\section{Foraging}
+
+\begin{definition} \label{def:Welfare}
+\textbf{Welfare} is the state of wealth, it can either be personal or collaborative (social).
+\end{definition}
+
+\begin{definition} \label{def:Maximising Social Welfare}
+\textbf{Maximising Social Welfare} is a strategy which attempts to maximise the total amount of welfare (resources) the islands have.
+\end{definition}
+
+
+\subsection{Foraging Background}
+
+Foraging is a fundamental requirement that is needed as it is a method of obtaining more resources from the starting amount. Foraging is based on the concept of inputting resources to get a return.
```

Comment:
Possibly wordy sentence and two sentences in succession starting with the same word. Suggested rewrite:

Foraging is a fundamental requirement required to obtain more resources during the simulation. It is based on the concept of inputting resources to get a return.